test-clients/src/main/java/com/hedera/services/legacy/file/FileServiceIT.java

```diff
@@ -9,9 +9,9 @@
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
- * 
+ *
```

Comment:
for this and other legacy test clients, do we still need them? I thought they are already migrated to EET based clients already.

Reply:
May need another discussion and can be done in a separate PR if we decide to remove them

Reply:
Yeah, agree.